android/java/org/chromium/chrome/browser/rewards/BraveRewardsCreatorPanelFragment.java

```diff
@@ -0,0 +1,355 @@
+            try {
+                mBannerInfo = new BraveRewardsBannerInfo(bannerInfoJson);
+            } catch (JSONException e) {
+                Log.e(TAG, "TippingBanner -> CreatorPanel:onAttach JSONException error " + e);
+            }
+        }
+    }
+
+    @Override
+    public View onCreateView(
+            LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.brave_rewards_creator_panel, container, false);
+        init(view);
+
+        return view;
+    }
+
+    @Override
+    public void onDestroyView() {
+        super.onDestroyView();
+        if (mIconFetcher != null) {
+            mIconFetcher.detach();
+        }
+
+        if (null != mBraveRewardsNativeWorker) {
+            mBraveRewardsNativeWorker.RemoveObserver(this);
+        }
+    }
+
+    private void init(View view) {
+        mBraveRewardsNativeWorker = BraveRewardsNativeWorker.getInstance();
+        mBraveRewardsNativeWorker.AddObserver(this);
+        mBraveRewardsNativeWorker.GetExternalWallet();
+
+        checkForShowSocialLinkIcons(view);
+        showVerifiedIcon(view);
+        retrivePublisherName(view);
+        retriveFavIcon();
+        setDescription(view);
+        setTitle(view);
+    }
+
+    private void checkForShowSocialLinkIcons(View view) {
+        if (mBannerInfo == null) return;
+        HashMap<String, String> links = mBannerInfo.getLinks();
+        if (links != null) {
+            for (Map.Entry<String, String> element : links.entrySet()) {
+                String name = element.getKey();
+                String url = element.getValue();
+                showSocialLinkIcon(view, name, url);
+            }
+        }
+    }
+
+    private void showSocialLinkIcon(View view, String name, String url) {
+        switch (name) {
+            case TWITTER:
+                ImageView twitterIcon = view.findViewById(R.id.twitterButton);
+                showAndSetListener(twitterIcon, url);
+                break;
+
+            case YOUTUBE:
+                ImageView youtubeIcon = view.findViewById(R.id.youtubeButton);
+                showAndSetListener(youtubeIcon, url);
+                break;
+
+            case TWITCH:
+                ImageView twitchIcon = view.findViewById(R.id.twitchButton);
+                showAndSetListener(twitchIcon, url);
+                break;
+            case GITHUB:
+                ImageView githubIcon = view.findViewById(R.id.githubButton);
+                showAndSetListener(githubIcon, url);
+                break;
+
+            case REDDIT:
+                ImageView redditIcon = view.findViewById(R.id.redditButton);
+                showAndSetListener(redditIcon, url);
+                break;
+
+            case VIMEO:
+                ImageView vimeoIcon = view.findViewById(R.id.vimeoButton);
+                showAndSetListener(vimeoIcon, url);
+                break;
```

Comment:
add a default case `default: // Do nothing` otherwise it will error out on chromium 106 build when modified